source/write/run-command.txt

```diff
@@ -0,0 +1,74 @@
+.. _java-rs-run-command:
+
+======================
+Run a Database Command
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: database operation, code example
+
+Overview
+--------
+
+In this guide, you can learn how to run a database command with the
+{+driver-short+}. Database commands can perform a variety of administrative and
+diagnostic tasks, such as fetching server statistics, initializing a replica
+set, or running an aggregation pipeline. You can use database commands when the
+driver does not contain a helper method for the command you want to run.
+
+.. important:: Prefer Driver Methods Over Database Commands
+
+   The driver provides methods for many database commands. We recommend
+   using driver methods instead of executing database commands when possible. 
+
+   To perform administrative tasks, use the `MongoDB Shell
+   <https://www.mongodb.com/docs/mongodb-shell/>`__ instead of the
```

Comment:
S: use the shell role https://github.com/mongodb/snooty-parser/blob/5dab215d105fa01976c6521c356323725e137a9d/snooty/rstspec-consolidated.toml#L1359C7-L1361